install/kubernetes/helm/istio/files/injection-template.yaml

```diff
@@ -11,7 +11,7 @@ initContainers:
 {{- end }}
   args:
   - "-p"
-  - "{{ .MeshConfig.ProxyListenPort }}"
+  - "15001"
```

Comment:
Is this hardcoded port correct? (looks like a test value cut-paste error to me)

Reply:
Yes this is intentional. We intentionally made a choice to not support customizable proxy listen port, this option is not tested, documented, or something we want to support. I am pretty sure it would not work either. Removing this allows us to not depend on MeshConfig at all in the injection template so it reduces complexity